servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/GrpcClientBuilder.java

```diff
@@ -281,9 +281,26 @@
      *
      * @return {@code this}
      * @see #unresolvedAddressToHost(Function)
+     * @deprecated Use {@link #hostHeaderFallback(boolean)}.
      */
+    @Deprecated
     public abstract GrpcClientBuilder<U, R> disableHostHeaderFallback();
```

Comment:
As we discussed offline, let's also add default implementations for the abstract methods we deprecate?